common/metadata_utils.py

```
@@ -64,8 +70,14 @@ def filter_out(v2docs, operator, value):
                     try:
                         if isinstance(input, list):
                             input = input[0]
```

Ruff (0.15.21)

[error] Variable `input` is shadowing a Python builtin

(A001)